package.json

```diff
@@ -49,7 +49,7 @@
     "prettier": "^2.7.1",
     "terser-webpack-plugin": "^5.3.3",
     "testdouble": "^3.16.6",
-    "ts-jest": "^28.0.5",
+    "ts-jest": "^29.1.1",
```

Comment:
`npm` was not happy installing an older version of `ts-jest` with a newer version of `jest`. Bumping appeased it.